lib/components/artwork_grids/generic_grid.js

```diff
@@ -10,6 +10,7 @@ import Artwork from './artwork'
 class GenericArtworksGrid extends React.Component {
   state: {
     sectionDimension: number,
+    sectionCount: number,
   };
 
   static defaultProps = {
```

Comment:
I guess we should make `sectionCount` in the props configurable for portrait and landscape for it to even make sense to be a prop. For now, though, maybe we should just remove the prop, as I don’t believe it’s used anywhere anyways.